config/routes.rb

```diff
@@ -398,6 +398,13 @@
         post 'fetch_testers'
       end
     end
+
+    resources :extensions do
```

Comment:
We can set a good precedent here by only using the default routes that we want. That's probably just `:create`, `:update`, and `:delete` (note: this is after you've implemented the changes to the data format I mentioned above).